src/Engine/Manager/PhysicsManager.hpp

```diff
@@ -35,6 +36,12 @@ class PhysicsManager {
          */
         void Update(float deltaTime);
         
+        /// Update transforms of entities according to positions of physics
+        /// components.
```

Comment:
Do you need 2 rows for this?

Reply:
Mostly for ourselves to not make lines too long. Doxygen output has no line break so it's cool.